src/services/send-audit-events.ts

```diff
@@ -21,26 +21,29 @@ const sqsClient = tracer.captureAWSv3Client(
 /**
  * Function that sends Audit Events to the TxMA Egress queue.
  * @param eventName - The event name used for sending off the event to identify the action taken.
- * @param userId - The User ID that is sent off in the event.
- * @param txmaExtensions - Information regarding interventions.
+ * @param eventEnum
```

Comment:
The linter is being a yap here, could we get a `-` character after the params, please?